js-variables-and-numbers_simple-calculator/js/index.js

```diff
@@ -0,0 +1,103 @@
+console.clear();
+
+let operand1 = 12;
+const operand2 = 4;
+
+/*
+Mathematical operations
+*/
+const addButton = document.querySelector('[data-js="add"]');
+const subtractButton = document.querySelector('[data-js="subtract"]');
+const multiplyButton = document.querySelector('[data-js="multiply"]');
+const divideButton = document.querySelector('[data-js="divide"]');
+const exponentButton = document.querySelector('[data-js="exponent"]');
+const moduloButton = document.querySelector('[data-js="modulo"]');
+
+addButton.addEventListener("click", () => {
+  // 1. Add the two operands and store the result in a variable. Log the variable's value to the console.
+  let result = operand1 + operand2;
+  console.log (result);
+});
+
+subtractButton.addEventListener("click", () => {
+  // 2. Subtract the two operands and store the result in a variable. Log the variable's value to the console.
+  let resultSubtraction = operand1-operand2;
+  console.log (resultSubtraction);
+});
+
+multiplyButton.addEventListener("click", () => {
+  // 3. Multiply the two operands and store the result in a variable. Log the variable's value to the console.
+  let resultMultiply = operand1*operand2;
+  console.log(resultMultiply);
+});
+
+divideButton.addEventListener("click", () => {
+  // 4. Divide the two operands and store the result in a variable. Log the variable's value to the console.
+  let divide=operand1/operand2;
+  console.log (divide);
+});
+
+exponentButton.addEventListener("click", () => {
+  // 5. Raise "operand1" to the value of "operand2" and store the result in a variable. Log the variable's value to the console.
+  let exponent = operand1**operand2;
+  console.log (exponent);
+});
+
+moduloButton.addEventListener("click", () => {
+  // 6. Calculate the remainder when dividing "operand1" by "operand2" and store the result in a variable. Log the variable's value to the console.
+  let remainder = operand1 % operand2;
+  console.log(remainder);
+});
+
+/*
+Update the first operand
+
+Please don't forget to think about the declaration of the variable "operand1".
+*/
+const increaseByOneButton = document.querySelector(
+  '[data-js="increase-by-one"]'
+);
+const increaseByFiveButton = document.querySelector(
+  '[data-js="increase-by-five"]'
+);
+const decreaseByOneButton = document.querySelector(
+  '[data-js="decrease-by-one"]'
+);
+const decreaseByFiveButton = document.querySelector(
+  '[data-js="decrease-by-five"]'
+);
+const multiplyByTwoButton = document.querySelector(
+  '[data-js="multiply-by-two"]'
+);
+const divideByTwoButton = document.querySelector('[data-js="divide-by-two"]');
+
+increaseByOneButton.addEventListener("click", () => {
+  // 7. Increase the value of "operand1" by 1. Log the new value to the console.
+  operand1+=1;
+  console.log(operand1);
+});
+
+increaseByFiveButton.addEventListener("click", () => {
+  // 8. Increase the value of "operand1" by 5. Log the new value to the console.
+  console.log(operand1+5);
```

Comment:
This works but it doesn't increase the value of the operand, so I'd do something like: operand1+=5

Reply:
Makes sense, thanks!

Reply:
Corrected.